src/Configuration.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * Italia PagoPA Proxy
+ * Cittadinanza Digitale PagoPA services
+ */
+
+// App configuration
+export const CONFIG = {
+  // PagoPa-Proxy Configuration
+  CONTROLLER: {
+    PORT: 3000,
```

Comment:
the configuration should be provided via env vars, see [12 factor](https://12factor.net/) - or at least make it configurable per-environment

Reply:
Done. It will be available into next Push